nuxeo-runtime/nuxeo-runtime-aws/src/test/resources/OSGI-INF/test-aws-config.xml

```diff
@@ -2,7 +2,7 @@
 <component name="org.nuxeo.aws.core.test">
 
   <extension target="org.nuxeo.runtime.aws.AWSConfigurationService" point="configuration">
-    <configuration>
+    <configuration id="myconfig">
```

Comment:
Add some more configs.  We still need the config without an id, plus an extra one like:
```
    <configuration id="custom">
      <accessKeyId>CUSTOM_KEY_ID</accessKeyId>
      <secretKey>CUSTOM_SECRET_KEY</secretKey>
      <sessionToken>CUSTOM_SESSION_TOKEN</sessionToken>
      <region>CUSTOM_REGION</region>
    </configuration>
```